src/AndroidDebugLauncher/AndroidDebugLauncher.csproj

```diff
@@ -137,6 +137,10 @@
     </Content>
   </ItemGroup>
   <ItemGroup>
+    <ProjectReference Include="..\DebugEngineHost\DebugEngineHost.csproj">
```

Comment:
Do the launchers use reference the wait loop? Is that why they need these project references?